packages/wrangler/src/index.tsx

```diff
@@ -347,6 +347,18 @@ export async function main(argv: string[]): Promise<void> {
                 "utf-8"
               )
             );
+
+            await writeFile(
+              pathToPackageJson,
+              JSON.stringify({
+                ...JSON.parse(await readFile(pathToPackageJson, "utf-8")),
+                scripts: {
```

Comment:
if package.json previously existed, then this will blow away their previous scripts section.